terracumber_config/tf_files/local_mirror.tf

```diff
@@ -122,11 +123,16 @@ module "mirror" {
   customize_minima_file = "mirror/etc/minima-customize.yaml"
   synchronize_immediately = true
   disable_cron = true
+  image = "opensuse154o"
   volume_provider_settings = {
     pool = local.pool
     // uncomment next line to use existing snapshot as starting point
     //    volume_snapshot_id = data.aws_ebs_snapshot.data_disk_snapshot.id
   }
+  provider_settings = {
+    mac                = "52:54:00:ba:b7:98"
+    memory             = 4096
+  }
```

Comment:
```
grog# virsh list
(...)
 139   susemanager-4-3-aws-build-validation-mirror   en cours d’exécution
```
--> nice, but will that be set up automatically next time? I see no `name =` directive in this maint.tf file.